hack/helloworld.sh

```diff
@@ -1,33 +1,38 @@
 #!/usr/bin/env bash
 
-NAME=hello
-TARGET=${USER:-world}
+set -Eeuo pipefail
+
+# shellcheck disable=SC1091,SC1090
+source "$(dirname "${BASH_SOURCE[0]}")/lib/__sources__.bash"
+
+NAME="${NAME:-hello}"
+TARGET="${USER:-world}"
 
 # Create a sample Knative Service
 cat <<EOF | kubectl apply -f -
 apiVersion: serving.knative.dev/v1alpha1
 kind: Service
 metadata:
-  name: $NAME
+  name: ${NAME}
 spec:
   template:
     spec:
       containers:
         - image: gcr.io/knative-samples/helloworld-go
           env:
             - name: TARGET
-              value: $TARGET
+              value: ${TARGET}
+          readinessProbe:
+            httpGet:
+              path: /
```

Comment:
This change is unnecessary. Knative will add a default probe anyway.

Reply:
Is that a default probe? Or default one is socket based instead of http get?

Reply:
The default one is socket based, but it shouldn't matter in this case.